Automattic/wp-calypso
# Classic Block: Remove

#### Changes proposed in this Pull Request

With the removal of Gutenlypso, I don't think our `classic-block` is used anywhere anymore. (IIUC, the Gutenframe just uses wp-admin's 'regular' Classic Block).

#### Testing instructions

- Run Calypso, verify that the editor still works fine. Try inserting a Classic Block!
- `grep 'classic-block'` and verify there are no occurrences
